docset/windows/configci/New-CIPolicyRule.md

```diff
@@ -174,6 +174,29 @@ This command generates a publisher rule for the specific file named ci.dll.
 The file ci.dll is a kernel component.
 Therefore, the cmdlet generates both a kernel rule and a user mode rule.
 
+### Example 4: Specify a policy rule for a file or folder path string
+```
+PS C:\> New-CIPolicyRule -FilePathRule '.\temp\ConfigCITestBinaries\'
```

Comment:
This is a bad example. At least for me it is not resolving the dot-reference. Also, since you don't include a wildcard, this will only allow an executable file called "ConfigCITestBinaries\", which won't exist. I think you want to have a trailing wildcard or something.

Reply:
Fixed the example by adding a trailing wildcard.